Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -106,61 +106,76 @@ protected virtual bool HasContentAttribute
 
     void IBocColumnRenderer.RenderTitleCell (BocColumnRenderingContext renderingContext, in BocTitleCellRenderArguments arguments)
     {
+      ArgumentUtility.CheckNotNull("renderingContext", renderingContext);
+
+      if (arguments.CellID == null)
+        throw new ArgumentException("arguments.CellID is null", "arguments");
+
       RenderTitleCell(new BocColumnRenderingContext<TBocColumnDefinition>(renderingContext), arguments);
     }
 
-    protected virtual void RenderTitleCell (BocColumnRenderingContext<TBocColumnDefinition> renderingContext, in BocTitleCellRenderArguments arguments)
+    private void RenderTitleCell (
+        BocColumnRenderingContext<TBocColumnDefinition> renderingContext,
+        in BocTitleCellRenderArguments arguments)
+    {
+      var bocCellAttributeRenderingContext = new BocCellAttributeRenderingContext<TBocColumnDefinition>(renderingContext);
+      AddAttributesToRenderForTitleCell(bocCellAttributeRenderingContext, in arguments);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Th);
```

Comment:
strategic blank line
```suggestion
      var bocCellAttributeRenderingContext = new BocCellAttributeRenderingContext<TBocColumnDefinition>(renderingContext);
      AddAttributesToRenderForTitleCell(bocCellAttributeRenderingContext, in arguments);

      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Th);
```